tutorials/14-food-review-tone-analysis-and-response.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    for i in range(len(response_df)):
</pre><p>Why not just print the df?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
The response is a little long so I want to print out more. df print cuts out too much response.

Reply:
Please check https://colab.research.google.com/notebooks/data_table.ipynb see if it helps